test/functional/wallet_tokens_change_supply.py

```diff
@@ -138,7 +138,8 @@ async def async_test(self):
             assert_in("Too many decimals", err)
 
             # issue a valid token
-            token_id, err = await wallet.issue_new_token("XXX", 2, "http://uri", address, 'lockable')
+            number_of_decimals = random.randrange(1, 4)
+            token_id, err = await wallet.issue_new_token("XXX", number_of_decimals, "http://uri", address, 'lockable')
```

Comment:
Should `number_of_decimals = 0` be tested too or is it covered elsewhere?

Reply:
@OBorce Please address this in the next PR. I'm about to merge this.